### DISCUSSION TITLE
Creating a new panel with it's login url

### DISCUSSION THREAD

**Discussion by amoktar** [filamentphp/filament#9602] [General]
hello filament 
first, thanks for this great repo

I've added new panel using `php artisan make:filament-panel app`
How can I add its own login page like the one which created with the default panel 

this is the default admin
`/admin/login`

I need a new login page with the url `app/login`

thanks

**Comment by leandrocfe** [comment]
```php
$panel->login()
```

**Reply by amoktar** [reply]
That's awesome
Thanks @leandrocfe
